src/observe-tag.ts

```diff
@@ -120,18 +120,27 @@ export default (tagFunction: () => TramOneElement): TramOneElement => {
 
 				elementToGiveFocus = allActiveLikeElements[elementIndexToGiveFocus] as ElementPotentiallyWithSelectionAndFocus;
 				// also try to set the selection, if there is a selection for this element
-				if (elementToGiveFocus.setSelectionRange !== undefined) {
-					elementToGiveFocus.setSelectionRange(
-						removedElementWithFocusData.selectionStart,
-						removedElementWithFocusData.selectionEnd,
-						removedElementWithFocusData.selectionDirection
-					);
+				try {
+					if (elementToGiveFocus.setSelectionRange !== undefined) {
+						elementToGiveFocus.setSelectionRange(
+							removedElementWithFocusData.selectionStart,
+							removedElementWithFocusData.selectionEnd,
+							removedElementWithFocusData.selectionDirection
+						);
+					}
+				} catch (exception) {
+					// don't worry if we fail
+					// this can happen if the element has a `setSelectionRange` but it isn't supported
+					// e.g. input with type="range"
```

Comment:
It's slightly baffling that range inputs have a `setSelectionRange` method, but don't actually support it, it's kinda strange, but there's no real way to avoid this other than put a try-catch around it.

Reply:
We could potentially catch the specific exception, and throw other exceptions we don't expect (although honestly I don't think we want to throw any exception, this is a very non-critical path for the app).